ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -505,11 +506,17 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         }
         for item in items {
             if item.tag == BarButtonItemTag.dismiss.rawValue {
-                item.badgeValue = isOn ? "12345" : nil
+                let badgeValue = isOn ? "12345" : nil
+                let badgeAccessibilityLabel = isOn ? "\(badgeValue ?? "Zero") items" : nil
```

Comment:
I think that setting the accessibility label with "Zero items" while the badgeValue is nil is a bit misleading from the demo perspective.
I think you can create non-optional constants here and using the ternary operator to pass them or nil (depending on the isOn argument.

Reply:
But it will only call zero items if the switch is ON and still the badge value is nil and so it's not going to say zero items while actually running the demo but yes if we would do such thing that the badge value switch is on and still badge value is nil then it will call zero items. So, that's what I think but I guess your point is that we should be crashing it if that's the case or correct me If I am wrong.